WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Downloads/Download List/ProductDownloadDnD.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+import CoreServices
+import Yosemite
+
+/// A wrapper around ProductDownload, to make it compatible in using as Drag and Drop data source in an table view. Represents a ProductDownload entity.
+///
+public class ProductDownloadDnD: NSObject, Codable {
+    public let download: ProductDownload
+
+    /// ProductDownloadDnD initializer.
+    ///
+    public init(download: ProductDownload) {
+        self.download = download
+    }
+
+    /// Public initializer for ProductDownloadDnD
+    ///
+    required public convenience init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let download = try container.decode(ProductDownload.self, forKey: .download)
+        self.init(download: download)
+    }
+
+    public func encode(to encoder: Encoder) throws {
```

Comment:
`public` isn't needed if the class is in `WooCommerce` module. But since the class has been moved to `Networking` module, I believe `public` is needed now.